src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeBlockchain.cs

```diff
@@ -97,12 +98,12 @@ private Task InitBlockchain()
             _api.EthereumEcdsa = new EthereumEcdsa(_api.SpecProvider.ChainId, _api.LogManager);
             _api.TxPool = new TxPool.TxPool(
                 new PersistentTxStorage(_api.DbProvider.PendingTxsDb),
-                Timestamper.Default,
                 _api.EthereumEcdsa,
                 _api.SpecProvider,
                 _api.Config<ITxPoolConfig>(),
                 _api.StateProvider,
-                _api.LogManager);
+                _api.LogManager,
+                CreateTxPoolTxComparer());
```

Comment:
why not _api.TxPoolComparer as with everything else?

Reply:
not a gret fan of constructors and method calls on the constructor args list

Reply:
need to overwrite it in AuRa when configured. I don't want to expose private things in _api